test/public/runs/detail.test.js

```diff
@@ -464,4 +466,26 @@ module.exports = () => {
         expect(await page.$eval('.external-links a:nth-of-type(2)', ({ href }) => href))
             .to.equal('http://localhost:8082/?page=layoutShow&runNumber=108&definition=COMMISSIONING&pdpBeamType=cosmic&runType=PHYSICS');
     });
+
+    it('should display links to environment in ECS if run is running', async () => {
+        // Test for not running run
+        await goToPage(page, 'run-detail', { queryParameters: { runNumber: 104 } });
```

Comment:
No more goToPage except for new test suite.

Reply:
applied